src/components/Chat/BottomSheetAttachment.js

```diff
@@ -49,7 +49,7 @@ const BottomSheetAttachment = React.forwardRef((props, ref) => (
             <Text style={styles.textCard}>Take a Picture</Text>
           </View>
           {props.isLoadingUploadCamera ? (
-            <ActivityIndicator size="small" color={COLORS.white2} />
+            <ActivityIndicator size="small" color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the color of the `ActivityIndicator` has been changed from `COLORS.white2` to `COLORS.white`. Please verify that this color change aligns with your design guidelines and maintains good contrast for accessibility.



<!-- This is an auto-generated comment by OSS CodeRabbit -->